changelog/4350.breaking.rst

```diff
@@ -0,0 +1,7 @@
+Removed deprecated functions:
```

Comment:
As stated in the changelog readme, the removal of deprecated features are "removals" and not "breaking". My logic for this (or rationalisation) is that I think it's important to keep the list of breaking changes down to "things you *need* to deal with".

Reply:
Changed in a PR.